src/brs/http/GetAccount.java

```diff
@@ -18,6 +18,7 @@
 
   private final ParameterService parameterService;
   private final AccountService accountService;
+  private final String depreciationMessage = "For account assets use getAccountAssets. This will be removed after next hardfork";
```

Comment:
`depreciationMessage` -> `deprecationMessage`
After next hardfork -> in V3.0

Reply:
So our next fork after sodium can be called as V3.0?